### PR TITLE
Update website with 0.18.0 release

### PR DESCRIPTION
**Pull request by ekinanp**

### CODE REVIEW

File: docs/_tutorials/03_extending_wash/writing-an-external-plugin.md [hunk 1]
```diff
@@ -38,11 +38,18 @@ function print_entry_json() {
   if [[ -z "${attributes_json}" ]]; then
     attributes_json="{}"
   fi
+
```
Comment (MikaelSmith) [inline docs/_tutorials/03_extending_wash/writing-an-external-plugin.md]: This page doesn't appear to be formatting correctly in general.

Comment (ekinanp) [reply docs/_tutorials/03_extending_wash/writing-an-external-plugin.md]: I don't understand -- I can see line numbers and code, and copy-pasting the code is pretty easy.

Comment (MikaelSmith) [reply docs/_tutorials/03_extending_wash/writing-an-external-plugin.md]: Ok, I can't trust github's markdown rendering for this.

**Review by MikaelSmith — APPROVED** [review]

**Comment by ekinanp** [comment]
One sec, I'm adding a bit more to the attributes-metadata tutorial for plugin authors.

File: docs/_tutorials/03_extending_wash/writing-an-external-plugin.md (hunk 1 above)
Comment (ekinanp) [reply docs/_tutorials/03_extending_wash/writing-an-external-plugin.md]: Ah, gotcha. Yeah, should have checked this out from my master branch then. Apologies for that.

**Comment by ekinanp** [comment]
Updated.

**Review by MikaelSmith — APPROVED** [review]

**Pull request merged**